crates/swift-bridge-ir/src/bridged_type/bridgeable_result.rs

```diff
@@ -129,19 +154,32 @@ impl BuiltInResult {
         type_pos: TypePosition,
         types: &TypeDeclarations,
     ) -> String {
-        let convert_ok =
-            self.ok_ty
-                .convert_ffi_expression_to_swift_type("val.ok_or_err!", type_pos, types);
-        let convert_err =
-            self.err_ty
-                .convert_ffi_expression_to_swift_type("val.ok_or_err!", type_pos, types);
-
-        format!(
-            "try {{ let val = {expression}; if val.is_ok {{ return {ok} }} else {{ throw {err} }} }}()",
-            expression = expression,
-            ok = convert_ok,
-            err = convert_err
-        )
+        if let Some(zero_byte_encoding) = self.ok_ty.only_encoding() {
```

Comment:
Minor, can we have a single `format!` instead of 2?

```rust
let (ok, err) = if let Some(zero_byte_encoding) = self.ok_ty.only_encoding() {
   // ...
} else {
  // ...
}

            format!(
                "try {{ let val = {expression}; if val.is_ok {{ return {ok} }} else {{ throw {err} }} }}()",
            )
```

Reply:
Done